src/services/DFGVariable.jl

```diff
@@ -46,7 +46,7 @@ getSofttype(dfg::AbstractDFG, lbl::Symbol, solvekey::Symbol=:default) = getSoftt
 
 Alias for [`getSofttype`](@ref).
 """
-getVariableType(params...) = getSofttype(params...)
+getVariableType(args...) = getSofttype(args...)
```

Comment:
perhaps?

```julia
getVariableType(args...; kwargs...) = getSofttype(args...; kwargs...)
```

Reply:
I'll have to look, This have been here for a while.

Reply:
It doesn't take keyword arguments, so should be fine